web.config

```diff
@@ -9,7 +9,7 @@
     <httpPlatform processPath="%JAVA_HOME%\bin\java.exe"
```

Comment:
I think you may have problems upgrading the app insights jar by removing the old one in the same PR
I have vague memories of windows file locking problems...

You can try this approach, but I believe the way that previously worked was:

1. add new jar and point web.config to it
2. separate PR remove old jar

Note that it will likely load the old code and not the new one during the transition period because of how it loads the code.

Reply:
Mmhhh not ideal. Let's see what happens. I'll add back the previous jar if it doesn't work